.github/workflows/release.yml

```diff
@@ -214,17 +214,17 @@ jobs:
                   name: contract-artifacts
 
             - name: Verify contracts
-              run: eval yarn hardhat --network ${{ needs.prepare.outputs.release_network }} ${{ needs.prepare.outputs.release_verify_cmd }} ${{ needs.prepare.outputs.release_version }}
+              shell: bash
```

Comment:
You can probably remove this ( default is bash anyway ) given it's not an action but directly running this step in the workflow. If it's action you will have to specify `shell: bash`. Let's try as-is and see if it suceed.

Reply:
You can ignore ^ this comment, this step succeeded